components/core/src/clp_s/Aggregation.cpp

```
@@ -0,0 +1,132 @@
+        // The tokenizer strips the namespace prefix out of the path, but namespaced fields live
+        // under a top-level object keyed by that namespace. Prepend it back so the path matches.
+        m_field_path.insert(m_field_path.begin(), descriptor_namespace);
+    }
+}
+
+auto MinMaxAggregation::beats_extreme(Extreme candidate) const -> bool {
+    auto const& current{m_extreme.value()};
+    if (m_find_max) {
+        return std::visit([](auto cand, auto cur) { return is_less(cur, cand); }, candidate, current);
+    }
+    return std::visit([](auto cand, auto cur) { return is_less(cand, cur); }, candidate, current);
```

Cppcheck (2.21.0)

[style] The function 'get_parent' is never used.

(unusedFunction)